### ISSUE TITLE
VirtualizerScrollView uses flex-direction for items

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#27285]
Using flex-direction to change orientation of items is an a11y antipattern since screen readers and keyboard navigation libraries will always follow DOM order. The change was introduced in https://github.com/microsoft/fluentui/pull/26985.

The usage of flex-direction should be removed because of a11y trap that users can fall into

**Comment by Mitch-At-Work** [comment]
This issue is ongoing, we have investigated aria-owns for providing 'true' order, however it is not a spec fully supported by the accessibility tools - we are investigating the possibility of an aria-reverse flag that would enable reverse order of components in the accessibility tree to align A11y with Flex CSS
